docs/stylesheets/star-rating.css

```diff
@@ -0,0 +1,143 @@
+@import url(//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css);
+
+
+#leave-feedback {
+  margin-top: 8px;
+}
+
+#starbox {
+  margin-top: -3px;
+}
+
+#stars {
+  display: block;
+  width: 125px;
+}
+
+#submitform {
+  border-radius: 4px;
+  display: none;
+  -webkit-animation: fadeIn 1 s;
+  animation: fadeIn 1 s;
+  position: absolute;
+  align-content: right;
+  width: 328px;
+  right: 0;
+  text-align: center;
+  background: white;
+  border: 1px solid rgba(131, 131, 131, 0.555);
+  z-index: 1;
+}
+
+#margin-starbox {
+  margin-right: 20px;
+}
+
+
+
+
+/* Add animation (fade in the popup) */
+@-webkit-keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+}
+
+/* Add animation (fade in the popup) */
+@-webkit-keyframes fadeIn {
```

Comment:
`@-webkit-keyframes fadeIn` and also `@keyframes fadeIn` seems to be declared twice?